crates/net/eth-wire-types/src/upgrade_status.rs

```diff
@@ -0,0 +1,48 @@
+//! Implement BSC upgrade message which is required during handshake with other BSC clients, e.g.,
+//! geth.
+
+use alloy_rlp::{RlpDecodable, RlpEncodable};
+use reth_codecs_derive::add_arbitrary_tests;
+
```

Comment:
can we add an id for the variant.

for encoding and decoding we can probably use `RawCapabilityMessage`

see `RawCapabilityMessage::eth` for ref